### PR TITLE
NF-82 - Access to service not allowed - message shown to users

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/controllers/ServiceUnavailableController.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import scala.concurrent.Future
+
+@Singleton
+class ServiceUnavailableController @Inject() (
+  mcc: MessagesControllerComponents,
+  unavailablePage: ServiceUnavailablePage
+) extends FrontendController(mcc) {
+
+  val onPageLoad: Action[AnyContent] = Action.async { implicit request =>
+    Future.successful(Ok(unavailablePage()))
```
Comment (crosslandwa) [inline app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/controllers/ServiceUnavailableController.scala:33]: OBSERVATION: This should pre-compute the `Ok(unavailablePage())` response if we want to use `Future.successful` (or use `Future()`)

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/controllers/actions/IdentifierAction.scala [hunk 2]
```diff
@@ -65,10 +65,7 @@ class AuthenticatedIdentifierAction @Inject() (
             internalId => block(IdentifierRequest(request, internalId, EoriNumber(eoriNumber)))
           ).getOrElse(throw new UnauthorizedException("Unable to retrieve internal Id"))
         else
-          Future.successful(
-            // TODO - crete new page for "not allowed"
-            Redirect(routes.UnauthorisedController.onPageLoad())
-          )
+          Future.successful(Redirect(routes.ServiceUnavailableController.onPageLoad()))
```
Comment (crosslandwa) [inline app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/controllers/actions/IdentifierAction.scala:68]: OBSERVATION: `Future()` / `Future.successful` usage here as well

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/controllers/ServiceUnavailableController.scala (hunk 1 above)
Comment (SteveSugden) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/controllers/ServiceUnavailableController.scala:33]: In actual fact, this doesn't need to be an async response.

**Review by crosslandwa — DISMISSED** [review]
LGTM - some minor observations about `Future/Future.successful` usage

**Review by crosslandwa — APPROVED** [review]

**Pull request merged**